libs/ml/include/ml/dataloaders/code2vec_context_loaders/context_loader.hpp

```diff
@@ -81,6 +82,12 @@ class C2VLoader : public DataLoader<LabelType, DataType>
    */
   ContextTensorsLabelPair GetNext() override;
 
+  /**
```

Comment:
function comments should go with the function implementation (even where the implementation is just to throw runtime error)